actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb

```diff
@@ -0,0 +1,74 @@
+require 'active_support/core_ext/object/try'
+
+module ActionView
+  module CollectionCaching
+    extend ActiveSupport::Concern
+
+    included do
+      # Use Rails.cache or the fallback cache store if Action View is used without Rails.
+      mattr_accessor(:collection_cache) do
+        Rails.try(:cache) || ActiveSupport::Cache::MemoryStore.new
```

Comment:
I don’t think we should be hanging off globals in ActiveSupport. Rails is the global context for this stuff.

> On Feb 16, 2015, at 11:22, Josef Šimánek <notifications@github.com> wrote:
> 
> In actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb:
> 
> > @@ -0,0 +1,74 @@
> > +require 'active_support/core_ext/object/try'
> > +
> > +module ActionView
> > +  module CollectionCaching
> > +    extend ActiveSupport::Concern
> > +
> > +    included do
> > +      # Use Rails.cache or the fallback cache store if Action View is used without Rails.
> > +      mattr_accessor(:collection_cache) do
> > +        Rails.try(:cache) || ActiveSupport::Cache::MemoryStore.new
> 
> What about to move Rails.cache to ActiveSupport.cache (and all configs and stuff) and just delegate it from Rails.cache there? So this line will change to ActiveSupport.cache and default storage will be chosen there?
> 
> Does it make sense? Would be this PR accepted?
> 
> —
> Reply to this email directly or view it on GitHub.
>